packages/react/src/components/TreeView/TreeView.tsx

```diff
@@ -148,6 +225,7 @@ export default function TreeView({
           }
         }
       }
+      // @ts-ignore
```

Comment:
What error is being suppressed here?

Reply:
Here I'm suppressing the same error as before
![image](https://github.com/carbon-design-system/carbon/assets/25573926/adc081c7-1159-499d-a603-04746a68c3af)